speechbrain/tokenizers/SentencePiece.py

```diff
@@ -224,11 +224,12 @@ def _csv2text(self):
                 )
                 break
             row_idx += 1
-            sent = row[index_label]
-            if self.char_format_input:
-                (sent,) = merge_char([sent.split()])
-                sent = " ".join(sent)
-            text_file.write(sent + "\n")
+            if len(row) > index_label:
```

Comment:
This seems to be a silent skip for empty CSV lines or lines which have missing columns. Feels like this should rather be fixed at the CSV creation stage, right? I feel like a silent skip should maybe only happen for empty lines. But this is a minor comment.